docs/index.rst

```diff
@@ -12,13 +12,21 @@
 TARDIS Core Package Documentation
 ##################################
 
+|
+
+.. raw:: html
+
+    <p style="text-align:center"><a href="http://tardis-sn.github.io/"><span style="font-size:36px">Click here for the TARDIS collaboration website</span></a></p>
+
+|
+
```

Comment:
This is not looking good and is too alarming. I would advise against it because contextually the homeage of tardis package should be about it only, atleast the 1st paragraph. 

How about we create a section after Mission Statement called "About the TARDIS Project" and in description say something like: 
> tardis package is the core package provided by the TARDIS Project. To learn more about the complete project, other packages it provides, and the community behind it, please check the [project website](https://tardis.github.io).

If we want to draw more attention than a hyperlink, I'm fine with also putting it in sidebar - potentially below API (I think other projects also do it, where some items in sidebar link to a separate website).